Gruntfile.js

```diff
@@ -1370,6 +1371,17 @@ module.exports = function(grunt) {
 		}
 	} );
 
+	grunt.registerTask( 'blockJson2PHP', 'Copies block.json file contents to block-json.php.', function() {
```

Comment:
`block.json` files are copied with webpack:
https://github.com/WordPress/wordpress-develop/blob/24b83dd431179c79b7fdba3080e04da8c3e50b0c/tools/webpack/blocks.js#L107-L114

I think it would be better to convert them to PHP files in one go. At the moment, you would have both `block.json` and `block-json.php` files in the same folder where the former becomes unused.

Reply:
We could easily remove the JSON files, I'm just not sure if it's something we want to do or not.
* I was thinking that _maybe_ we want to use the `.json` files instead of the `.php` files when `WP_DEBUG` and/or `SCRIPT_DEBUG` is `true`
* The concept in this patch was to make this change as minimal as possible, avoiding any backwards-compatibility issues etc, just in case a plugin is accessing the JSON files for core blocks. Is that a valid worry, or do you think it's an edge-case? 🤔 

Also, I'm not that familiar with `webpack` so using `grunt` was a crutch, the only way I knew how to do this 😅